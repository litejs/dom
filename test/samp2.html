<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<h1 data-x="&lt;&amp;a&#32;&#39;&#x27;&ac;">Hello World 2</h1>
<script defer src='/beacon.min.js' data-cf-beacon='{"version":"2022.11.3","token":"5c2880733a3347458426171ca8b905db"}'></script>
<table cellpadding="0" cellspacing="0" border="0"><!-- table --!>
	<tbody><tr>
			<td>A</td>
			<td>B</td>
	</tr></tbody>
</table>

