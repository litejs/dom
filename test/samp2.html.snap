<h1 data-x="&lt;&amp;a ''&amp;ac;">Hello World 2</h1><script defer src="/beacon.min.js" data-cf-beacon='{"version":"2022.11.3","token":"5c2880733a3347458426171ca8b905db"}'></script>